x/wasm/keeper/relay.go

```diff
@@ -131,8 +131,11 @@ func (k Keeper) OnRecvPacket(
 	if execErr != nil {
 		return nil, sdkerrors.Wrap(types.ErrExecuteFailed, execErr.Error())
 	}
+	if res.Err != "" { // handle error case as before https://github.com/CosmWasm/wasmvm/commit/c300106fe5c9426a495f8e10821e00a9330c56c6
```

Comment:
Yes, I agree with these changes.
Current contracts shouldn't hit this branch, but if they do, they behave the same as before the update.